src/test/templates.ts

```diff
@@ -1,3 +1,8 @@
+export const template = '<html>\n<head>\n<style>\n.page {\npadding-left: 10px;\npadding-right: 10px;\npage-break-after: auto;\n}\n ' +
+  '</style>\n</head>\n<body>\n<div id="container" class="page">\n<p style="text-align:center;">Enrollment #: <span>{{ enrollmentId }}' +
+  '</span></p>\n<p><span class="char-style-override-2" style="line-height: 1.2;">Course Certificate</span></p>\n</body>\n</html>';
```

Comment:
This one would definitely be easier to read with backticks and template expressions :-)
See my previous comment for an example.

Reply:
Great idea to make my "template" a template expression.  Using that approach I was able to mock an entire template which is readable.